packages/quantic/force-app/main/default/lwc/quanticModal/quanticModal.js

```diff
@@ -94,4 +103,51 @@ export default class QuanticModal extends LightningElement {
   get tabindex() {
     return this.isVisible ? 0 : -1;
   }
+
+  get allAssignedElements() {
+    const headerSlot = this.template.querySelector('slot[name=header]');
+    const contentSlot = this.template.querySelector('slot[name=content]');
+    const footerSlot = this.template.querySelector('slot[name=footer]');
+
+    const allAssignedElements = [
+      // @ts-ignore
+      ...headerSlot.assignedElements(),
+      // @ts-ignore
+      ...contentSlot.assignedElements(),
+      // @ts-ignore
+      ...footerSlot.assignedElements(),
+    ].map((element) => Array.from([element, ...element.childNodes]));
+    return this.flattenArray(allAssignedElements);
+  }
+
+  focusOnLastElement() {
+    for (const element of this.allAssignedElements.reverse()) {
+      const lastFocusableElement = getLastFocusableElement(element);
```

Comment:
You're probably missing a type or using the wrong type if `.focus()` is not working no?

Reply:
Correct! I just fixed it. getLastFocusableElement was returning Element instead of HTMLElement.